QEfficient/cloud/infer.py

```diff
@@ -60,6 +60,7 @@ def main(
     device_group: List[int] = [
         0,
     ],
+    execute : bool = True
```

Comment:
this funtionality exists in `QEfficinet/cloud/execute.py::main`, not required to add here.